include/igl/opengl/ViewerData.cpp

```diff
@@ -265,24 +263,25 @@ IGL_INLINE void igl::opengl::ViewerData::set_texture(
 }
 
 IGL_INLINE void igl::opengl::ViewerData::set_data(
-  const double caxis_min,
-  const double caxis_max,
-  const Eigen::VectorXd & D)
+  const Eigen::VectorXd & D,
+  double caxis_min,
+  double caxis_max,
+  igl::ColorMapType cmap)
 {
   if(!show_texture)
   {
     Eigen::MatrixXd CM;
-    igl::parula(Eigen::VectorXd::LinSpaced(21,0,1).eval(),false,CM);
+    igl::colormap(cmap,Eigen::VectorXd::LinSpaced(256,0,1).eval(),0,1,CM);
     set_colormap(CM);
-  } 
+  }
   set_uv(((D.array()-caxis_min)/(caxis_max-caxis_min)).replicate(1,2));
 }
 
-IGL_INLINE void igl::opengl::ViewerData::set_data( const Eigen::VectorXd & D)
+IGL_INLINE void igl::opengl::ViewerData::set_data( const Eigen::VectorXd & D, igl::ColorMapType cmap)
```

Comment:
Let's keep a `set_data(D)` that uses a default colormap. 

We should also use `const` for input params.

Reply:
nvm about default. Found the default it in the .h